### PR TITLE
Fix compatibility with `release-it` v15

### LINKED ISSUE #203: Incompatible with release-it v15
```
ERROR Package subpath './lib/util.js' is not defined by "exports" in /Users/jhawker/Projects/GitHub/ember-resize-modifier/node_modules/release-it/package.json imported from /Users/jhawker/Projects/GitHub/ember-resize-modifier/node_modules/release-it-lerna-changelog/index.js
```

This error is reproducible in #201:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './test/util/index.js' is not defined by "exports" in /home/runner/work/release-it-lerna-changelog/release-it-lerna-changelog/node_modules/release-it/package.json imported from /home/runner/work/release-it-lerna-changelog/release-it-lerna-changelog/test.js
```

### PR DESCRIPTION
**Pull request by bertdeblock**
This will probably fail on the Node v12 scenario.

Closes #203.

### CODE REVIEW

**Comment by rwwagner90** [comment]
@bertdeblock have you tried this out and it works? Might use your branch here until this is merged.

**Comment by bertdeblock** [comment]
@rwwagner90 I used the `--dry-run` flag to manually test these changes, but I didn't do an actual release.
Everything went smooth when performing a dry run, so I don't expect any hiccups when doing a real release.
All tests were passing as well locally. If you do use this branch, feel free to ping me if you run into any issues.

File: test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import { createRequire } from 'module';
 import tmp from 'tmp';
 import test from 'ava';
-import { factory, runTasks } from 'release-it/test/util/index.js';
+import { factory, runTasks } from 'release-it/test/util';
```
Comment (scalvert) [inline test.js:6]: The existing import should still work. Why was this change necessary? (release-it's exports contain `test/util/index.js`)

Comment (scalvert) [reply test.js:6]: Oh I see what's happened. `release-it`'s exports alias the path, but that's not a valid path for esm. I'll see if I can get it to work.

Comment (scalvert) [reply test.js:6]: So the issue that remains is that you can't use directory imports in ESM, but the full path is not exposed via `release-it`'s exports field. I think this issue may need to be resolved in `release-it` itself.

Comment (scalvert) [reply test.js:6]: So it looks like changing the `exports` field within `release-it` will fix the issue. I'll open a PR there.

Comment (scalvert) [reply test.js:6]: Opened an issue (https://github.com/release-it/release-it/issues/920).

Comment (scalvert) [reply test.js:6]: That issue is resolved and a new verison released. I'll update this PR accordingly.

Comment (scalvert) [reply test.js:6]: Aaaaand the fix didn't work as the path aliased is incorrect. I've opened a [new PR](https://github.com/release-it/release-it/pull/921) in `release-it` to fix the path.

Comment (scalvert) [reply test.js]: I've worked around this issue for now, somewhat heavy handedly for this PR, by converting the test harness to `jest` over `ava`. This also serves the purpose of matching the test infra with the sibling `release-it-yarn-workspaces` plugin. Most importantly, it unblocks shipping this package.

**Comment by bertdeblock** [comment]
Thanks for finishing this @scalvert! I was on vacation.

**Comment by scalvert** [comment]
My pleasure, @bertdeblock! Hope your vacation was good!

**Pull request merged**